device/tests/Microsoft.Azure.Devices.Client.Test/IoTHubClientDiagnosticTest.cs

```diff
@@ -0,0 +1,188 @@
+﻿namespace Microsoft.Azure.Devices.Client.Test
+{
+    using System;
+    using System.Collections.Specialized;
+    using System.Globalization;
+    using System.IO;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Web;
+    using Microsoft.Azure.Amqp;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class IoTHubClientDiagnosticTest
+    {
+        [TestMethod]
+        [TestCategory("IoTHubClientDiagnostic")]
```

Comment:
I don't think the category is needed - please remove. We can easily filter by executing only tests within this class.

Reply:
Because our test cases are in DeviceClientTest.cs and IoTHubClientDiagnosticTest.cs. Using category can help us quickly test our tests.

![image](https://user-images.githubusercontent.com/5545529/32037080-fcebf758-ba55-11e7-9db6-4d760d931c1f.png)